pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -607,6 +607,7 @@ func (o *MustGatherOptions) newPod(node, image string) *corev1.Pod {
 		},
 		Spec: corev1.PodSpec{
 			NodeName:      node,
+			PriorityClassName: "system-cluster-critical",
```

Comment:
can you provide the comment you gave me in slack about the three options, what they mean and why I chose this one?

As I recall, I wanted this pod to be oomkillable, but not pre-emptable.